instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-jersey-2.0/javaagent/src/main/java/org/glassfish/jersey/client/ResponseCallbackWrapper.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.glassfish.jersey.client;
+
+import static io.opentelemetry.javaagent.instrumentation.jaxrsclient.v2_0.JerseyClientUtil.handleException;
+
+import javax.ws.rs.ProcessingException;
+import org.glassfish.jersey.process.internal.RequestScope;
+
+// implemented interface is package private so wrapper needs to be in the same package
+public class ResponseCallbackWrapper implements ResponseCallback {
```

Comment:
so that it's more obvious that this is a class we are injecting into this package, and doesn't look like a regular jersey class
```suggestion
public class OpenTelemetryResponseCallbackWrapper implements ResponseCallback {
```